uaclient/security.py

```diff
@@ -378,14 +377,23 @@ def cves(self) -> List[CVE]:
     def title(self):
         return self.response.get("title")
 
+    @property
+    def references(self):
+        return self.response.get("references")
+
     def get_url_header(self):
         """Return a string representing the URL for this notice."""
-        lines = [
-            "{issue}: {title}".format(issue=self.id, title=self.title),
-            "Found CVEs:",
-        ]
-        for cve in self.cves_ids:
-            lines.append("https://ubuntu.com/security/{}".format(cve))
+        lines = ["{issue}: {title}".format(issue=self.id, title=self.title)]
+
+        if self.cves_ids:
+            lines.append("Found CVEs:")
+            for cve in self.cves_ids:
+                lines.append("https://ubuntu.com/security/{}".format(cve))
+        elif self.references:
```

Comment:
I guess they can, but my understanding is that we should only list those if we have no CVEs, because the CVE itself already contains references to those bug links

Reply:
Sounds good to me!